client/src/components/Icons/SVGIcon.css

```diff
@@ -0,0 +1,20 @@
+.logoNav {
+	background-color: #fae5dd;
+	list-style: none;
+	color: black;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-around;
+	padding: 20px;
+	margin: 10px;
+}
```

Comment:
We don't want to have CSS like this that's just to make this render in Storybook in a certain way. We only want CSS that's related to the icon itself